src/main/java/com/minecolonies/coremod/commands/colonycommands/AddOfficerCommand.java

```diff
@@ -101,8 +112,25 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
             playerName = sender.getName();
         }
 
+        executeShared(server, sender, colony, playerName);
+    }
+
+    private void executeShared(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final Colony colony, final String playerName)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "server". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)